frontend/13f-app/src/components/CompanyFilingList.css

```
@@ -0,0 +1,32 @@
+#filings-list {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 80%;
+    margin-top: 25px;
+    margin-left: 25px;
+}
+
+#filings-list h1 {
+    padding-bottom: 50px;
+}
+
+#filings-list table {
+    align-self: center;
+}
+
+#filings-list td, #filings-list th {
+    border: 5px solid #ddd;
+    padding: 8px;
+}
+  
+#filings-list tr:nth-child(even) {
+    background-color: #f2f2f2;
+}
+  
+#filings-list th {
+    padding-top: 12px;
+    padding-bottom: 12px;
```

Use the [padding shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding)
```suggestion
    padding: 12px 0 12px 0;
```